src/gui/wxgui/MainWindow.cpp

```diff
@@ -1771,6 +1760,42 @@ void MainWindow::SetFullScreen(bool state)
 		SetMenuVisible(true);
 }
 
+void MainWindow::EndEmulation() // unfinished - memory leaks and crashes after repeated use (after 3x usually)
+{
+	CafeSystem::ShutdownTitle();
+	DestroyCanvas();
+	m_game_launched = false;
+	m_launched_game_name.clear();
+	#ifdef ENABLE_DISCORD_RPC
+	if (m_discord)
+		m_discord->UpdatePresence(DiscordPresence::Idling, "");
+	#endif
+
+	if (GetConfig().disable_screensaver)
+		ScreenSaver::SetInhibit(false);
+
+	// close memory searcher if created
+	if (m_toolWindow)
+	{
+		m_toolWindow->Close();
+		m_toolWindow = nullptr;
+		m_memorySearcherMenuItem->Enable(false);
+	}
+
+	// unsure about this, but seems to free up additional 10 MB memory, see LatteThread.cpp
+	if (g_renderer) 
```

Comment:
Cleaning up the renderer is already handled by `LatteThread_Exit()` which in turn is triggered by `Latte_Stop()` in `ShutdownTitle()`